README.md

```diff
@@ -823,7 +823,7 @@ For ex:
 }
 ```
 
-#### POST /api/v2/alias/custom/new
+#### POST /api/v3/alias/custom/new
```

Comment:
@ntnhon this new endpoint version uses `mailboxes` as input to create multiple mailboxes for the new alias.

Reply:
@nguyenkims I would suggest saying `mailbox_ids` instead of `mailboxes` to be consistent with `PUT /api/aliases/:alias_id`

Reply:
Good idea! I just updated the PR.